blockchain/check-backup.js

```diff
@@ -102,7 +102,7 @@ stream.on("finish", async () => {
 });
 
 const loadConfig = (path) => {
-  const config = yaml.safeLoad(fs.readFileSync(path, "utf8"));
+  const config = yaml.load(fs.readFileSync(path, "utf8"));
```

Comment:
pls do not replace `load` by` safeLoad` more information [here](http://adilapapaya.com/docs/js-yaml/#safeloadstringoptions)

Reply:
this docs are deprecated (and not even versioned lol). 
The current version of js-yaml is 4.x.x, so this applies: 
https://github.com/nodeca/js-yaml/blob/master/migrate_v3_to_v4.md